.jshintignore

```diff
@@ -3,6 +3,7 @@ libs
 lib-jitsi-meet.js
 lib-jitsi-meet.min.js
 lib-jitsi-meet.js.map
+karma.conf.js
```

Comment:
Why do we need to exclude karma.conf.js?

webpack.config.js is not excluded for example, what is the difference?